src/RoomsSDKAdapter.js

```diff
@@ -36,42 +37,67 @@ export default class RoomsSDKAdapter extends RoomsAdapter {
     };
   }
 
+  /**
+   * Create an observable that subscribes to room change events emitted via websocket and
+   * emits the changed room object.
+   *
+   * @private
+   * @param {string} ID ID of room to get
+   * @returns {Observable.<Room>} Observable that emits changes to the room
+   * @memberof RoomsSDKAdapter
+   */
+  addRoomChangeListener(ID) {
+    const roomChange$ = fromEvent(this.datasource.rooms, ROOM_UPDATED_EVENT);
```

Comment:
We should standardize on the wording, `change` vs `update`. Personally, I prefer `update`